packaging/rpm/SPECS/apache-ignite.spec

```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
 Apache Ignite™ is the in-memory computing platform composed of a strongly
 consistent distributed database with powerful SQL, key-value and processing APIs
 
+This package is a virtual aggregation package for all packages for current
+Apache Ignite release - install it to install all other packages
+
+
+%package -n %{nameCore}
+Summary:          Apache Ignite execution core 
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Requires:         java-1.8.0, pkgconfig
+Requires(pre):    shadow-utils
+Provides:         %{name}-core
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameCore}
+Apache Ignite's core files and libs, necessary for node operation
+
 
+%package -n %{nameLibs}
+Summary:          Apache Ignite optional libs
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Requires:         %{name}-core = %{version}-%{release}
+Provides:         %{name}-libs
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameLibs}
+Apache Ignite's optional libs and integrations
+
+
+%package -n %{nameBenchmarks}
+Summary:          Apache Ignite benchmarks
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Requires:         %{name}-core = %{version}-%{release}
+Provides:         %{name}-benchmarks
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameBenchmarks}
+Prepared maven project for Apache Ignite's benchmarks conduct
+
+
+%package -n %{nameExamples}
+Summary:          Apache Ignite examples
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Provides:         %{name}-examples
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameExamples}
+Apache Ignite's integration and usage examples
+
+
+%package -n %{nameDocs}
+Summary:          Apache Ignite documentation
```

Comment:
Only saw scaladocs there, no javadocs. Is my build process broken?

Reply:
If built from correct sources, both `scaladoc` and `javadoc` exist in package. See (this build!)[https://ci.ignite.apache.org/viewLog.html?buildId=1169369&buildTypeId=Releases_NightlyRelease_ApacheIgniteNightlyReleaseAssembleRpmPackages&tab=artifacts] for example.